specs/examples/README.md

```diff
@@ -0,0 +1,157 @@
+# Nimble Examples
```

Comment:
I'm gonna close this PR for now. Some things I think would be useful if re-opening:

- Split out the new cdn / bundle packages as separate work
- Consider completely narrowing the scope of the examples to just getting started examples. At which point maybe they should just create template packages:
  - `dotnet new install`: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new-install
  - `npm init`: https://docs.npmjs.com/cli/v10/commands/npm-init
  - Pros: folks don't have to sync down the repo to get started / figure out how to download just a specific folder (which GitHub doesn't make easy)